pages/bellapps.md

```diff
@@ -35,12 +35,13 @@ You will be shown the page below. Make sure you fill it out completely.
 ![Become an Administrator](uploads/images/become_admin.png)
 
 Next, you will need to fill out your configurations. Make sure you write your name all in lowercase letters and your code all in uppercase letters, as in the example below:
```

Comment:
Should I put the "note" like this one?
![image](https://cloud.githubusercontent.com/assets/26332034/24574176/4512d6c4-1654-11e7-882d-2ce4bb1731c3.png)